File: kimMinsik/week1.js
//최종풀이
function solution(n) {
  let tmp = [1, 2];
  for (let i = 2; i < n; i++) {
    tmp.push((tmp[i - 1] + tmp[i - 2]) % 1000000007);
  }

  return tmp[n - 1];
}

File: 차도희/week1.java
class Solution {
    public int solution(int n) {
        int[] arr = new int[n+1];
        arr[0] = 1;
        arr[1] = 2;
        int index = 2;
        while(index < n)
        {
            arr[index] = (arr[index-1] + arr[index-2]) % 1000000007;
            index++;
        }
        
        return (arr[n-1]);
    }
}